include/diagnostics/scaling_ratio.hpp

```diff
@@ -0,0 +1,87 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 
+
+// Contributed and/or modified by Iva Janković, as part of Google Summer of Code 2025 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef DIAGNOSTICS_SCALING_RATIO_HPP
+#define DIAGNOSTICS_SCALING_RATIO_HPP
+
+template<typename Polytope, typename MT>
+std::pair<std::vector<double>, typename Polytope::MT>scaling_ratio_boundary_test(const Polytope&           P,
+                                                                                const MT&                 samples,
+                                                                                const std::vector<int>&   facet_id,
```

Comment:
here we assume that the facet_id already contains the facet id for each sample. Do we have a function to compute it? If not, why not computing it inside the function and do not take it as an input?